### ISSUE TITLE
Newbie question about step 7

### ISSUE THREAD

**Issue by vanillameringue** [Soreepeong/XivMitmLatencyMitigator#94]
Hello! Sorry for having an amateur question but I am a total newbie with coding and linux but since I play on console I really want to make this work but I keep encountering hiccups.

-I have  followed the steps from 1-6 so far and everything seems to be going smoothly but I encountered an issue with step 7 which leads to my confusion:
![IMG_0397](https://github.com/Soreepeong/XivMitmLatencyMitigator/assets/165493275/c1313e67-3f7a-4e59-8ca2-bf4f4c50d1d3)

If I am understanding this correctly, I need to drag and drop ffxiv.exe or ffxiv_dx11.exe into the vm. My confusion is the next part, download run HFS. Initially I did this inside the linux os vm but even with Wine installed I can’t open and run HFS which leads me to assume that I should download and run it on the host os? would really help me if I can receive help and advice on this. I use a macbook to do this though and my linux os is Fedora so I don’t know if it has anything to do with it. I want to know if HFS needs to be installed and run on linux vm? and if so why does it not want to open even with wine installed.

**Comment by Soreepeong** [comment]
If you're using Linux already, then you don't have to use HFS; it's purely for the purpose of transferring ffxiv exe from Windows PC to a linux environment. This step really is about copying the executable file into the directory.

**Comment by bankjaneo** [comment]
You can also use SCP to copy file between macOS and Linux

`scp SOURCE DESTINATION`

`scp "/path/to/your/exe/on/mac/ffxiv_dx11.exe" root@VM_IP:/path/you/want/to/keep/file/`

Replace root if you use other username.

**Comment by vanillameringue** [comment]
Hello! I was happy that I managed to make some progress but I encountered an issue that I didn’t see in the other posts. I tried to curl but it came out with Segmentation Fault (Core Dumped).
![IMG_8391](https://github.com/Soreepeong/XivMitmLatencyMitigator/assets/165493275/6a5597a5-4fad-4b1a-a4a0-11a1c4b1ff9b)

**Comment by bankjaneo** [comment]
Can you show the output of following command?

`python --version`

**Comment by vanillameringue** [comment]
Of course!

![IMG_8393](https://github.com/Soreepeong/XivMitmLatencyMitigator/assets/165493275/465f4474-e514-44c5-86fb-9d4cae2cf87e)

**Comment by bankjaneo** [comment]
Python version does not seems to be a problem. However, I'm not a Fedora expert, but from quick research on `Segmentation fault (core dumped)` error. It looks like some python modules is not working correctly. You may want to completely uninstall python and install it again. But as mentioned, I'm no Fedora expert, I don't know how to do that or if it will solve your problem.

Or you may try create a Debian/Ubuntu VM for XivMitmLatencyMitigator instead.

**Comment by vanillameringue** [comment]
Hello! Yes! I tried to run it on both ubuntu and debian and both came out with similar error but instead of Segmentation Fault its Illegal Instruction. I’m beginning to wonder if it has something to do with apple silicon since it seems there’s compatibility issues with it.

![IMG_8394](https://github.com/Soreepeong/XivMitmLatencyMitigator/assets/165493275/7801835a-8172-4a64-96b3-0f0abe8ef2b9)

**Comment by bankjaneo** [comment]
That's definitely the source of problem! You need x64 VM to run the script. The ARM VM for Apple Silicon will not work.

**Comment by vanillameringue** [comment]
Hello! Thank for your advice! I took in what you said and tried something out: Instead of virtualization I tried to instead do emulation via UTM vm for mac. I managed to download and run ubuntu x64 even though its slow the important thing was I could run the terminal. Here’s the result:
![IMG_8404](https://github.com/Soreepeong/XivMitmLatencyMitigator/assets/165493275/adad1a08-e67b-411a-90b3-585e4c989c92)

It didn’t reply with an error so I hope I’m on the correct path. I didn’t test it out on ff yet cause I’m still unsubbed.
